spec/factories/institutions.rb

```diff
@@ -18,13 +18,15 @@
     approved { true }
     bad_address { false }
     high_school { false }
+    ownership_name { nil }
```

Comment:
Is there a reason why this isn't ownership_name { 'test' } instead of redefining this variable with the same value?

Reply:
Its just because of the test, it looks at shared_examples_for_exportable_by_version which compares a step that creates a new instance of institution and compares it to whats being imported from the csv. The ownership field is updated in another process so that is why it is nil.